modules/nf-core/mosdepth/main.nf

```diff
@@ -2,7 +2,7 @@ process MOSDEPTH {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::mosdepth=0.3.3"
+    conda "${moduleDir}/environment.yml"
```

Comment:
Mosdepth had two releases since and is now on 0.3.5 ok for me to update later but just to tonte it down somewhere

Reply:
https://github.com/nf-core/modules/issues/4289